python/cuml/tests/test_umap.py

```diff
@@ -27,8 +27,10 @@
 from sklearn.cluster import KMeans
```

Comment:
Do we want to add `build_algo` as a parameter for all the tests? It looks like it could increase the testing time. But, I don't know might be necessary.

Reply:
You are right! I am planning to add a table the contains the results of real benchmark datasets (mnist, sift, gist, kaggle million headlines). I believe I have done enough investigation to verify that this is working well..! So I changed the tests test for just a few for minimum verification.